Dockerfile

```diff
@@ -23,6 +23,7 @@ RUN apk upgrade --update && \
 # install codebase
 WORKDIR "${LOCALDIR}"
 RUN npm install --unsafe-perm
+RUN npm install leakage
```

Comment:
for some reason leakage instalation fails if i attemd to make it global -g flag. Complains about some user privileges even if you add USER root or node or docker... So i leaved it local.

Reply:
We should add this to `package.json` so that folks don't have to perform this step when doing development.  That is, `leakage` will just be imported when one runs an `npm install` in the repo root.  This also means the step right above `RUN npm install --unsafe-perm` would take care of the install of `leakage`.